react/Carousel.js

```diff
@@ -51,22 +45,33 @@ class Carousel extends Component {
   }
 
   render() {
-    const { banner1, banner2, banner3, banner4 } = this.props
+    const {
+      height,
+      mobileHeight,
+      banner1,
+      banner2,
+      banner3,
+      banner4,
+      banner5,
+      banner6,
+    } = this.props
     const { loading } = this.state
     const settings = this.configureSettings()
 
-    const banners = [banner1, banner2, banner3, banner4]
+    const banners = [banner1, banner2, banner3, banner4, banner5, banner6]
 
     return (
-      <div className="vtex-carousel pa7">
+      <div className="vtex-carousel">
         {!loading && (
           <Slider {...settings}>
             {banners.map(function(banner, i) {
               if (banner && banner.image) {
                 return (
-                  <div key={i}>
+                  <div key={i} style={{ maxHeight: `${height}px` }}>
```

Comment:
Use something else as key.
https://medium.com/@robinpokorny/index-as-a-key-is-an-anti-pattern-e0349aece318